Nodejs/Product/Nodejs/TestFrameworks/Tape/tape.js

```diff
@@ -0,0 +1,75 @@
+var fs = require('fs');
+var path = require('path');
+
+function find_tests(testFileList, discoverResultFile, projectFolder) {
+    var test = findTape(projectFolder);
+    if (test === null) {
+        console.error("NTVS_ERROR:", "Couldn't find 'tape' module relative to", projectFolder);
+        return;
+    }
+    
+    var harness = test.getHarness({ exit: false });
+    var tests = harness["_tests"];
+
+    var count = 0;
+    var testList = [];
+    testFileList.split(';').forEach(function (testFile) {
+
+        var testCases;
+        process.chdir(path.dirname(testFile));
+        try {
+            testCases = require(testFile);
+        } catch (ex) {
+            console.error("NTVS_ERROR:", ex, "in", testFile);
+            return;
+        }
```

Comment:
[nit] Since this logic is duplicated, please extract it to it's own function that returns a Boolean that indicates if the test was loaded or not.